### PR TITLE
gVXR2CIL

### PR DESCRIPTION
**Pull request by effepivi**
## Describe your contribution

A data reader from gVXR to CIL.

## Checklist when you are ready to request a review

- [X ] I have performed a self-review of my code
- [ X] I have created a new folder, containing my contributions which are in the form of jupyter notebooks, with any necessary supporting python files, and a LICENSE file.
- [ X] I have added a description of my contribution(s) to the top of my file(s)
- [ X] If publicly available, I have added a link to the dataset used near the top of my file(s)
- [X ] I have added the CIL version I ran with near the top of my file(s)
- [ X] The content of this Pull Request (the Contribution) is intentionally submitted for inclusion in CIL-User-Showcase.
- [ X] I confirm that the contribution does not violate any intellectual property rights of third parties.
- [ X] I confirm that I have added license headers to all of the files I am contributing (with a license of my choice)
- [ X] Change pull request label to 'Waiting for review' 

Note: for an example of a contribution, where a license header, description, data link and CIL version has been added, please
see: [example_contribution](https://github.com/TomographicImaging/CIL-User-Showcase/blob/main/example_contribution)

### CODE REVIEW

**Comment by effepivi** [comment]
Hi @MargaretDuff 

thanks. 

I was careful not to store 3rd party files on the GitHub repository. It's downloaded and unziped on the fly (see Cell 16 of the code).

I got it from [https://gate.uca.fr/download/examples-tools](https://gate.uca.fr/download/examples-tools). There is a small PDF in the ZIP file. This is from the pEdiatRic dosimetRy personalized platfORm (ERROR) ([https://error.upatras.gr/](https://error.upatras.gr/)). P. Papadimitroulas et al., "A Review on Personalized Pediatric Dosimetry Applications Using Advanced Computational Tools," in IEEE Transactions on Radiation and Plasma Medical Sciences, vol. 3, no. 6, pp. 607-620, Nov. 2019, doi: [10.1109/TRPMS.2018.2876562](https://doi.org/10.1109/TRPMS.2018.2876562). As far as I can tell the license is not specified. we'll need to contact the authors of the paper or the project coordinator.

> [effepivi#1](https://github.com/effepivi/CIL-User-Showcase/pull/1)
> 
> Thanks for a really interesting use of CIL. I made some changes and opened the above pull request.
> 
> @effepivi It would be good to be clear in the document where the data for the child phantom came from and what licenses etc are relevant. Do you have that information? Perhaps instead of storing the required data in the git repository, it could be put on zenodo?

**Comment by MargaretDuff** [comment]
After a discussion with @effepivi, he will update the notebook with the new version of gvxr. Thanks!

File: gVXR2CIL/CBCT-acquisition-with-noise-and-reconstruction.ipynb
Comment (MargaretDuff) [inline gVXR2CIL/CBCT-acquisition-with-noise-and-reconstruction.ipynb:1]: A few quick changes:
- can you add a line under import packages that points to the environment file you provided
- Can you add the CIL version number somewhere near the top of the file. e.g "The data is reconstructed with CIL version 23.0.1" in the 2nd cell
- Can you add a few more words about where you got the data from before you download and extract the ZIP file, I think you said "I got it from https://gate.uca.fr/download/examples-tools. There is a small PDF in the ZIP file. This is from the pEdiatRic dosimetRy personalized platfORm (ERROR) (https://error.upatras.gr/). P. Papadimitroulas et al., "A Review on Personalized Pediatric Dosimetry Applications Using Advanced Computational Tools," in IEEE Transactions on Radiation and Plasma Medical Sciences, vol. 3, no. 6, pp. 607-620, Nov. 2019, doi: [10.1109/TRPMS.2018.2876562](https://doi.org/10.1109/TRPMS.2018.2876562)."
- Can you add a line to explain the sliders - how should the user interact with them? 
-

**Review by MargaretDuff — COMMENTED** [review]
Looks good!

Please can you also rename your gVXR2CIL folder with the name 006_gVXR2CIL to match with the other contributions.

File: gVXR2CIL/CBCT-acquisition-with-noise-and-reconstruction.ipynb
Comment (effepivi) [reply gVXR2CIL/CBCT-acquisition-with-noise-and-reconstruction.ipynb:1]: done

**Review by jakobsj — CHANGES_REQUESTED** [review]
The notebook looks truly amazing, an impressive complete workflow and what looks like the start of very nice integration between gVXR and CIL!

Just a few minor comments, as I didn't check all the details:
- I think I saw a previous version with all images and output in, but now it looks empty. We want to add the pictures and output right?
- There is a lot going on in the notebook but it is well structured, so it is easy to navigate.  I was thinking if the files in the folder could perhaps be organised similarly , for example all pictures in one folder e.g. "img/" and any data in eg "data/". Right now, one sees all the files at once and need to look a bit to identify which one is contains the demo to be looked at.

**Comment by MargaretDuff** [comment]
Thanks @effepivi! It is working with gvxr=2.0.7. I am happy to merge it in

**Review by MargaretDuff — APPROVED** [review]

**Comment by effepivi** [comment]
> Approving with just the two small comments:
> 
> Under main steps, between bullets no 5. and 6. there is a broken image or something in the beginning of the line.
> 
> islicer output is not shown, just HBox something, not sure whether to do anything about it though.

All the interactive plots are not visible, e.g. between Steps 5 and 6 or the islicers. If you know a fix, I'll be interested. Surely it's also an issue with other notebooks?

**Comment by jakobsj** [comment]
Yeah, the islicers I could see, but I didn't know what the one between 5.
and 6. was, I just see some broken image icon.  Happy for you to merge!

**Comment by effepivi** [comment]
> Yeah, the islicers I could see, but I didn't know what the one between 5. and 6. was, I just see some broken image icon. Happy for you to merge!

How do I do that?

**Comment by MargaretDuff** [comment]
I can merge - thanks @effepivi and @jakobsj!

**Pull request merged**